### ISSUE TITLE
Logging out inside the Prefect base image incorrectly breaks the CLI

### ISSUE THREAD

**Issue by kvnkho** [PrefectHQ/prefect#5647]
## Description

Exec into the prefect image (this is currently 1.1) with 
```
docker run -it prefecthq/prefect
```
and login with
```
prefect auth login --key API_KEY
```
and then logout
```
prefect auth logout
```
and run the logout again
```
prefect auth logout
```
and you will get:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/prefect/client/client.py", line 293, in tenant_id
    tenant_id = uuid.UUID(tenant_id)
  File "/usr/local/lib/python3.7/uuid.py", line 160, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string
```
which is seemingly fine but then when you login again, you get:
```
  File "/usr/local/lib/python3.7/site-packages/prefect/client/client.py", line 293, in tenant_id
    tenant_id = uuid.UUID(tenant_id)
  File "/usr/local/lib/python3.7/uuid.py", line 160, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string
```
if fact, even
```
prefect auth list-tenants
```
yield the same error.

Prefect diagnostics inside the container:
```
{
  "config_overrides": {},
  "env_vars": [],
  "system_information": {
    "platform": "Linux-5.10.25-linuxkit-x86_64-with-debian-11.2",
    "prefect_backend": "cloud",
    "prefect_version": "1.1.0",
    "python_version": "3.7.12"
  }
}
```

**Comment by madkinsz** [comment]
This should be fixed by #5643 in 1.2.0

**Comment by bendizen** [comment]
How can we fix this in 1.1.0 ?
